mdata/aggmetric.go

```diff
@@ -186,22 +186,36 @@ func (a *AggMetric) GetAggregated(consolidator consolidation.Consolidator, aggSp
 			if agg == nil {
 				return Result{}, fmt.Errorf("consolidator %q not configured", consolidator)
 			}
+
+			// note: only the raw AggMetric potentially has a ROB
+			// these children aggregator AggMetric never do, so result.Points is never set here
 			result, err := agg.Get(from, to)
 			if err != nil {
 				return Result{}, err
 			}
-			futurePoints := []schema.Point{}
+
+			var futurePoints []schema.Point
```

Comment:
@zlin179 FYI in Go it's valid, and recommended to use the zero value, nil. this saves an allocation. it can be safely iterated and appended to.